### DISCUSSION TITLE
update progress does not work..

### DISCUSSION THREAD

**Discussion by valeriomatranga** [askdkc/livewire-csv#70] [General]
hello, when importing large files, the import progress update does not work! it seems that the data is passed only on initialization.

**Comment by askdkc** [comment]
@valeriomatranga 

Check your .env file and make sure it has

`QUEUE_CONNECTION=database`


Also run

`php artisan queue:work`

https://laravel.com/docs/10.x/queues#the-queue-work-command

😉
